__tests__/hooks/useGroupInfo.test.js

```diff
@@ -35,6 +34,7 @@ jest.mock('recoil', () => ({
 }));
 
 describe('useGroupInfo should run correctly', () => {
+  afterEach(cleanup);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good addition of `afterEach(cleanup)`. This will ensure that after each test, if the component is unmounted, any remaining tasks that would run during this phase are cancelled. This improves the reliability of tests.
```diff
- beforeEach(() => {
+ afterEach(cleanup);
  beforeEach(() => {
    jest
      .spyOn(servicePermission, 'requestExternalStoragePermission')
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->